README.md

```diff
@@ -80,7 +80,7 @@ For example (from [example-pom.xml](src/test/resources/example-pom.xml)):
                 <eventConfig implementation="io.perfana.event.wiremock.WiremockEventConfig">
                     <name>WiremockEvent1</name>
                     <wiremockFilesDir>src/test/resources/wiremock-stubs</wiremockFilesDir>
-                    <wiremockUrl>http://localhost:9999</wiremockUrl>
+                    <wiremockUrl>http://localhost:9999/__admin/mappings</wiremockUrl>
```

Comment:
are mappings and settings same type of files? we can also use subdirs in wiremock-stubs that translate into the sub-urls to call?

Reply:
They are both json files :-)

Reply:
it would even be more nice if you could use multiple endpoints during your schedule